writer/tests/unit/core/test_writer_service.py

```diff
@@ -109,7 +109,7 @@ def test_writer_single_thread(writer):
     writer.current_lock = 0
     writer.position = 0
 
-    def wait_for_lock():
+    def wait_for_lock(*args, **kwargs):
```

Comment:
what do you do with these args?

Reply:
Nothing, but I think mypy complained because `writer.write_with_database_context` now takes arguments, so this method had to take them as well.